tips/TIP-0026/indexer-rest-api.yaml

```diff
@@ -0,0 +1,778 @@
+            type: integer
+          example: 1000
+          description: Return outputs that are timelocked after a certain milestone index.
+        - in: query
+          name: hasExpirationCondition
+          schema:
+            type: boolean
+          example: true
+          description: Filters outputs based on the presence of expiration unlock condition.
+        - in: query
+          name: expiresBefore
+          schema:
+            type: integer
+          example: 1643383242
+          description: Return outputs that expire before a certain Unix timestamp.
+        - in: query
+          name: expiresAfter
+          schema:
+            type: integer
+          example: 1643383242
+          description: Return outputs that expire after a certain Unix timestamp.
+        - in: query
+          name: expiresBeforeMilestone
+          schema:
+            type: integer
+          example: 1000
+          description: Return outputs that expire before a certain milestone index.
+        - in: query
+          name: expiresAfterMilestone
+          schema:
+            type: integer
+          example: 1000
+          description: Return outputs that expire after a certain milestone index.
+        - in: query
+          name: expirationReturnAddress
+          schema:
+            type: string
+          example: iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx
+          description: Filter outputs based on the presence of a specific return address in the expiration unlock
+            condition.
+        - in: query
+          name: sender
+          schema:
+            type: string
+          example: iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx
+          description: Filters outputs based on the presence of validated Sender (bech32 encoded).
+        - in: query
+          name: tag
+          schema:
+            type: string
+          example: 4ec7f23
+          description: Filters outputs based on matching Tag Block.
+        - in: query
+          name: createdBefore
+          schema:
+            type: integer
+          example: 1643383242
+          description: Return outputs that were created before a certain Unix timestamp.
+        - in: query
+          name: createdAfter
+          schema:
+            type: integer
+          example: 1643383242
+          description: Return outputs that were created after a certain Unix timestamp.
+        - in: query
+          name: pageSize
+          schema:
+            type: integer
+          example: 10
+          description: The maximum amount of items returned in one call. If there are more items, a cursor to the next page is returned too.
+        - in: query
+          name: cursor
+          schema:
+            type: string
+          example: 0c78e998f5177834ecb3bae1596d5056af76e487386eecb19727465b4be86a790200.10
+          description: Starts the search from the cursor (confirmationMS+outputId.pageSize).
```

Comment:
Maybe add a comment that the `pageSize` query is ignored when the page size is defined via the cursor.